core/io/translation_loader_po.cpp

```diff
@@ -286,7 +286,7 @@ Ref<Resource> TranslationLoaderPO::load_translation(Ref<FileAccess> f, Error *r_
 
 			ERR_FAIL_COND_V_MSG(end_pos == -1, Ref<Resource>(), "Expected '\"' at end of message while parsing: " + path + ":" + itos(line));
 
-			l = l.substr(0, end_pos);
+			l = l.left(end_pos);
```

Comment:
`end_pos` is the result of find() and ERR_FAIL_COND_V_MSG() above discards the -1 case.